include/sway/desktop/fx_renderer/fx_renderer.h

```diff
@@ -33,6 +46,8 @@ struct decoration_data {
 	float *dim_color;
 	bool has_titlebar;
 	bool blur;
+	bool can_blur_xray;
```

Comment:
We should be able to tell if a layer surface can be x-rayed with it's layer height + if the x-ray setting is enabled, don't think we need this variable

Reply:
That's true! This was added before we passed the layer pointer to the function 😅